lesson2/index.js

```diff
@@ -0,0 +1,19 @@
+const ul = document.getElementById("js-target");
+
+const fragment = document.createDocumentFragment();
+const a = document.createElement("a");
+a.href = "1.html";
+
+const img = document.createElement("img");
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
+const text = document.createTextNode("これです");
```

Comment:
IMO
You can write this
and
A class name such as `anchor` instead of `a` might be easier to understand.

```suggestion
a.textContent = "これです";
```

Reply:
I learned a lot! Thank you.